src/content/docs/apm/apm-ui-pages/monitoring/external-services/external-services-setup.mdx

```diff
@@ -0,0 +1,235 @@
+</Callout>
+
+Here is some information about the different routines for adjusting the reservoir for APM agents, as well as tips for adjusting sampling for OpenTelemetry.
+
+If you need additional help with adjusting sampling, please see these [support options](https://docs.newrelic.com/docs/licenses/license-information/general-usage-licenses/global-technical-support-offerings/).
+
+### APM agent sampling [#sampling-agents]
+
+<Callout variant="tip">
+These steps don't apply to classic external services.
+</Callout>
+
+All APM agents have a reservoir that stores spans, and most of these agent reservoirs are configurable. The size of this reservoir affects the likelihood that an agent is able to send all the spans that it creates. See [fragmented traces](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui/#fragmented-traces) for more details. 
+
+
+Data for this feature is derived from adjacent client and server spans where external calls are made from one service to another. When an agent hits its span reservoir limit, there's a chance it will drop some spans representing these calls.
+
+The default agent configuration value of 2000 defines the maximum number of events the agent collects per minute. If there are more spans than this number, the agent collects a statistical sampling, essentially degrading the amount of data that is used in the external services map.
+
+If you’re not seeing the type of detail you want in the UI, you can increase the size of the reservoir up to 10,000. Review the following to adjust the agent reservoirs:
+
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        APM agent
+      </th>
+      <th>
+        Documentation
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        C SDK
+      </td>    
+      <td>
+       The reservoir is not currently configurable 
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Go
+      </td>    
+      <td>
+       The reservoir is not currently configurable
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Java
+      </td>    
+      <td>
+       [Java configuration](/docs/apm/agents/java-agent/configuration/java-agent-configuration-config-file/#cfg-span-events-max-samples-stored) 
+      </td>
+    </tr>
+    <tr>
+      <td>
+        .NET
+      </td>    
+      <td>
+        [.NET configuration](/docs/apm/agents/net-agent/configuration/net-agent-configuration/#paragrp-max-samples-stored) 
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Node.js
+      </td>    
+      <td>
+        [Node.js configuration](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration/#span-events-max-samples-stored) 
+      </td>
+    </tr>
+    <tr>
+      <td>
+        PHP
+      </td>    
+      <td>
+       The reservoir is not currently configurable
```

Comment:
Add period

Reply:
@barbnewrelic, same as above.

Reply:
ignore, i got carried away in my own little world of sentences not column consistency :-)